Package.swift

```diff
@@ -34,3 +34,13 @@ let package = Package(
     ),
   ]
 )
+
+//for target in package.targets {
+//  target.swiftSettings = target.swiftSettings ?? []
+//  target.swiftSettings?.append(
+//    .unsafeFlags([
+//      "-Xfrontend", "-warn-concurrency",
+//      "-Xfrontend", "-enable-actor-data-race-checks",
+//    ])
+//  )
+//}
```

Comment:
Should it be enabled?

Reply:
If we do, SPM complains about unsafe flags being used when integrating it in other packages/projects

Reply:
ohh, that's a good point. Just thinking loud here. What if we introduce ENV varible for local development and CI.
Something like :

```swift
if ProcessInfo.processInfo.environment["COMPOSABLE_DEVELOPER"] == "1" {
settings.append(.unsafeFlags([
     "-Xfrontend",
     "-warn-concurrency",
     "-Xfrontend",
     "-enable-actor-data-race-checks"
     ]))
}
```

Little incovinience to define variable locally but it can be documented on the README for developers.

Reply:
That's definitely an interesting idea! Let's see what the rest have to say 🤓